manifests/tftp.pp

```diff
@@ -44,7 +44,9 @@
     ensure_packages(['wget'], { ensure => $wget_version, })
   }
 
+  class { 'foreman_proxy::tftp::netboot':
```

Comment:
This is the only change I didn't understand. What is the purpose of this one?

Reply:
Previously the `$root` parameter was inherited, but could be overridden. I first tried to test just this class, but found it wasn't very useful since it depends on various directories created by the `tftp` class. It was easy to miss this dependency because when reading the parameters, most, but not all are inherited from its own params class. I hope that this makes it a bit clearer.